src/index.html

```diff
@@ -72,5 +82,27 @@
         </div>
       </div>
     </div>
+  <script type="text/javascript">
+    // From https://css-tricks.com/snippets/javascript/get-url-variables/
+    function getQueryVariable(variable) {
+      var query = window.location.search.substring(1);
+      var vars = query.split("&");
+      for (var i=0;i<vars.length;i++) {
+        var pair = vars[i].split("=");
+        if(pair[0] == variable){return pair[1];}
+      }
+      return(false);
+    }
+
+    var containerElem = document.getElementById("recommend-channel-container");
```

Comment:
at a glance, I'm not sure what `containerElem` is - sounds like it might be for the whole page? 
maybe it could be called `channelContainerElem`